ee/tabby-schema/src/schema/web_documents.rs

```diff
@@ -12,17 +12,17 @@ pub struct CustomWebDocument {
     pub name: String,
     pub id: ID,
     pub created_at: DateTime<Utc>,
+    pub updated_at: DateTime<Utc>,
     pub job_info: JobInfo,
 }
 
 #[derive(GraphQLObject)]
 #[graphql(context = Context)]
 pub struct PresetWebDocument {
-    pub name: String,
```

Comment:
need to keep `name` field - from user's perspective of view, they treat `id` as opaque string, without knowing its content.

Reply:
done